en/computer_vision/obstacle_avoidance.md

```diff
@@ -160,6 +171,12 @@ The array waypoints contains all NAN expect for index 0:
 The setpoints are tracked by the multicopter position controller.
 
 
+## Testing
```

Comment:
FYI moved this to end. I suspect this is no longer true - ie we use a different platform. Do we have any detailed instructions for what people can do to duplicate our hardware setup?

Reply:
We have a list of tested companion computers and cameras in the Avoidance repository readme

Reply:
Thanks. I'll just link then.